services/user/Homepage/ui/src/apps/chainmail/components/compose-dialog.tsx

```diff
@@ -0,0 +1,389 @@
+                            <FormField
+                                control={form.control}
+                                name="message"
+                                render={({ field }) => (
+                                    <FormItem className="flex flex-1 flex-col">
+                                        <FormControl>
+                                            <Textarea
+                                                placeholder="Message"
+                                                className="h-full resize-none text-sm sm:min-h-[200px]"
+                                                {...field}
+                                                onChange={(e) => {
+                                                    field.onChange(e);
+                                                    updateDraft();
+                                                }}
+                                            />
+                                        </FormControl>
+                                        <FormMessage />
+                                    </FormItem>
+                                )}
+                            />
+                        </div>
+                        <DialogFooter className="flex flex-col-reverse gap-2 pb-4 sm:flex-row sm:justify-between sm:space-x-2 sm:pb-0">
+                            <Button
+                                variant="outline"
+                                onClick={(e) => {
+                                    e.preventDefault();
+                                    onOpenChange(false);
+                                }}
+                                className="w-full sm:w-auto"
+                                type="button"
+                            >
+                                <X className="mr-2 h-4 w-4" />
+                                Cancel
+                            </Button>
+                            <AlertDialog>
+                                <AlertDialogTrigger asChild>
+                                    <Button className="w-full sm:w-auto">
+                                        <Send className="mr-2 h-4 w-4" />
+                                        Send Message
+                                    </Button>
+                                </AlertDialogTrigger>
+                                <AlertDialogContent>
+                                    <AlertDialogHeader>
+                                        <AlertDialogTitle>
+                                            Messages are not private
+                                        </AlertDialogTitle>
+                                        <AlertDialogDescription>
+                                            Chain Mail is for demonstration
+                                            purposes only. Messages are not
+                                            currently encrypted and are
+                                            permanently stored on a publicly
```

Comment:
I'd remove the word permanently, which sort of implies that they are stored in state